clients/tabby-agent/src/TabbyAgent.ts

```diff
@@ -140,6 +141,12 @@ export class TabbyAgent extends EventEmitter implements Agent {
     if (options.config) {
       await this.updateConfig(options.config);
     }
+    if (!this.config.anonymousUsageTracking.disable) {
+      await anonymousUsageLogger.event({
+        event: "Initialize Agent",
```

Comment:
i believe the signature matches https://posthog.com/docs/libraries/js#send-custom-events-with-posthogcapture